examples/pytorch/rect/README.md

```diff
@@ -0,0 +1,41 @@
+# **DGL Implementation of RECT (TKDE20)**
+
+This DGL example implements the GNN model **RECT** (or more specifically its supervised part **RECT-L**) proposed in the paper [Network Embedding with Completely-imbalanced Labels](https://ieeexplore.ieee.org/document/8979355). The authors' original implementation can be found [here](https://github.com/zhengwang100/RECT).
+

```

Comment:
Thanks for contributing to DGL! Maybe you can add your github information in README like: This example was implemented by Fizyhsp.